stable/kommander/templates/deployment.yaml

```diff
@@ -0,0 +1,68 @@
+---
```

Comment:
Can we move service account (roles and bindings) information to another file.

Reply:
Done